doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To implement a quantized model with
+nGraph, provide a partially (or fully) quantized FP32 model (e.g., where the
+convolution layer in the model is replaced with a quantized convolution) to the
+nGraph library along with quantized parameters: weights, activations, scale, and
+zero point. 
+
+.. Note:: Currently, nGraph only supports quantization for inference.
+
+nGraph Quantized Operators (Ops)
+--------------------------------
+
+nGraph uses scale and zero point (also used by ONNX) to map real values to
+quantized values.  All quantized ops (APIs) in nGraph use scale and zero point
+and they can be used just like any other nGraph op. 
+
+**Scale**: the quantization scale for the tensor 
+
+**Zero point**: the zero point of the tensor 
+
+**Round mode**: used in combination with scale and zero point to round real 
+values to quantized values
+
+.. table:: nGraph Quantized Ops
+
+
+	+----------------------+-----------------------------------------------+
+	| Operator             | Description                                   |
+	+======================+===============================================+
+	| Quantize             | Maps real values (r) to quantized values (q)  |
+	|                      | using scale (s), zero point (z),              |
+	|                      | and round mode; produces a quantized tensor   |
+	+----------------------+-----------------------------------------------+
+	| Dequantize           | Maps quantized values (q) to real values (r)  |
+	|                      | using scale (s) and zero point (z); converts  |
+	|                      | a quantized tensor to a floating point tensor |
+	+----------------------+-----------------------------------------------+
+	| FakeQuantize         | Performs element-wise linear quantization     |
+	+----------------------+-----------------------------------------------+
+	| QuantizedConvolution | Performs quantized convolution; takes the     |
+	|                      | takes the same parameters as the              |
+	|                      | non-quantized convolution operator            |
+	+----------------------+-----------------------------------------------+
+	| QuantizedDot         | Performs quantized dot; takes the same        |
+	|                      | parameters as the non-quantized dot operator  |
+	+----------------------+-----------------------------------------------+
+
+Some frameworks such as TensorFlow have fused (or layer) ops. nGraph provides
+optional operations (listed in the following table) to help users easily
+translate (map) any quantized model created from frameworks with fused ops to
+nGraph.
+
+.. table:: Experimental Quantized Ops (optional)
+
+
+	+-----------------------------------+----------------------------------+
+	| Operator                          | Description                      |
+	+===================================+==================================+
```

Comment:
Minor nit: you use sentence-casing (end with period) on the other table but not this one.